_docs/manage/features/index.md

```diff
@@ -16,6 +16,8 @@ storage features. No feature labels are present by default.
 | Caching     | `storageos.feature.nocache`     | true / false   | Switches off caching. |
 | Compression | `storageos.feature.nocompress`  | true / false   | Switches off compression of data at rest and in transit. |
 | Replication | `storageos.feature.replicas`    | integers [0, 5]| Replicates entire volume across nodes. Typically 1 replica is sufficient (2 copies of the data); more than 2 replicas is not recommended. |
+| Failure mode | `storageos.com/failure.mode`   | strings [`soft`,`hard`,`alwayson`] | Soft failure mode works together with the failure tolerance. Hard is a mode where any loss in desired replicas count will mark volume as unavailable. AlwaysOn is a mode where as long as master is alive volume will be writable. |
+| Failure tolerance | `storageos.com/failure.tolerance` | integers [0, 4] | Specifies how many failed replicas we tolerate, defaults to (Replicas - 1) if Replicas > 0, so if we have 2 replicas it will default to 1, if we have 3 replicas, tolerance will be 2. |
```

Comment:
"Specifies how many failed replicas we tolerate" => "Specifies how many failed replicas to tolerate".

Remove the examples here, this should just a short description.